### ISSUE TITLE
Support for Ariston devices

### ISSUE THREAD

**Issue by Viproz** [nbogojevic/midea-beautiful-air#27]
Hey everyone,

I've recently bought dehumidier from the brand Ariston and was surprised to find midea behind the smart side of the device.

The model of the device: Ariston Deos 16S NET.
The name of the manufacturer in the wifi router: GD Midea AirConditioning Equipment Co.,Ltd.

I'm trying to use this repo to be able to talk locally to the device but when I use the discover function it doesn't recognize my account:
`WARNING:midea_beautiful.cloud:Authentication error: '3102' - 'this account does not exist'`
I have tried with all three --app implemented.

I'm guessing since I'm supposed to use the application "Ariston Clima" on android that the accounts are separated. On the password reset page (https://mapp-iam.appsmb.com/v1/page/repwd?activeId=fra_x_yyyyy&appId=1005&lan=en_US&stamp=20231202191008816&sign=xxx) I can see this:
`<input type="hidden" id="appId" name="appId" value="1005" />`
So the appid is 1005 but no sign of appkey or signkey.

Can somebody find the appkey and signkey for the application Ariston Clima or tell me how I can extract this information?

**Comment by Viproz** [comment]
Actually I found it!
For future reference I downloaded the apk on my computer, uncompressed it with 7zip and then searched for different elements with "find in files" in Notepad++.

I can confirm the signkey still is xhdiwjnchekd4d512chdjx5d8e4c394D2D7S (was present in libsmart.so)
Serching with the regex [a-z0-9]{32} (case sensitive) I found what looked like a key in the file resources.arsc: 434a209a5ce141c3b726de067835d7f0

Discovering new devices with 
`midea-beautiful-air-cli discover --account "email" --password MySecurePassword --appkey 434a209a5ce141c3b726de067835d7f0 --appid 1005`
Works and returns my device.

I'll continue exploring and I will share my findings for a future implementation.
